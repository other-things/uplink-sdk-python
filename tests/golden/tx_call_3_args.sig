r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 72601647661157045178718190515607357437124749478916635467285097908424389537104
