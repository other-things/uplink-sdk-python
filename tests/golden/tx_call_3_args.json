{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": {
                            "contents": {
                                "decimalIntegerValue": 1,
                                "decimalPlaces": 0,
                                "tag": "Dec"
                            },
                            "tag": "NumDecimal"
                        },
                        "tag": "VNum"
                    },
                    {
                        "contents": {
                            "contents": {
                                "decimalIntegerValue": 1,
                                "decimalPlaces": 0,
                                "tag": "Dec"
                            },
                            "tag": "NumDecimal"
                        },
                        "tag": "VNum"
                    },
                    {
                        "contents": {
                            "contents": {
                                "decimalIntegerValue": 1,
                                "decimalPlaces": 0,
                                "tag": "Dec"
                            },
                            "tag": "NumDecimal"
                        },
                        "tag": "VNum"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE03MjYwMTY0NzY2MTE1NzA0NTE3ODcxODE5MDUxNTYwNzM1NzQzNzEyNDc0OTQ3ODkxNjYzNTQ2NzI4NTA5NzkwODQyNDM4OTUzNzEwNA=="
}